build/tasks/aria-supported.js

```diff
@@ -69,10 +70,11 @@ module.exports = function (grunt) {
       const destFile = this.data.destFile;
       // Format the content so Prettier doesn't create a diff after running.
       // See https://github.com/dequelabs/axe-core/issues/1310.
-      const formattedContent = format(content, destFile);
-
-      // write `aria supported` file contents
-      grunt.file.write(destFile, formattedContent);
+      format(content, destFile).then(formattedContent => {
+        // write `aria supported` file contents
+        grunt.file.write(destFile, formattedContent);
+        done();
+      });
```

Comment:
Can you please add a `.catch` at the end to make sure the build fails if either format or the file write fail?